lib/resolveModule.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+const path = require('path');
+
+
+module.exports = function resolveModule(workingDirectory, moduleName) {
+  const absolutePath = path.resolve(workingDirectory, moduleName);
+  return fs.existsSync(absolutePath) || fs.existsSync(`${absolutePath}.js`)
+    ? path.resolve(absolutePath)
```

Comment:
Why should we resolve already resolved path once more?

Reply:
That's a good question. Looks like a leftover from some furious changes in code. I'll play with the code for a while and get back to address this comment.

Reply:
Okay, verified it has no particular reason. Addressed in da645b2